scripts/prepare-environment-k3d.sh

```diff
@@ -121,6 +121,7 @@ else
       { "backend": { "service": { "name": "epinio-server", "port": { "number": 80 } } }, "path": "/exit", "pathType": "ImplementationSpecific" } }]'
   fi
 fi
+helm upgrade --install epinio -n epinio --version 1.6.1 --create-namespace epinio/epinio --set global.domain=172.19.0.3.omg.howdoi.website
```

Comment:
This looks like leftover debug code, with its fixed version and host.

Reply:
Never mind. Next commit removed it.

Reply:
yes. It was. 
thanks for checking